### PR TITLE
docs(installer): add ingress documentation for control plane helm chart

### PR DESCRIPTION
**Pull request by bradmccoydev**
This documentation is to update values of the helm chart of the Keptn Control Plane (installer)

Closes https://github.com/keptn/keptn/issues/5297

Related PR to keptn/keptn: PR  keptn/keptn#5315

@mowies @thisthat 

Also updated the mongodb example to say enabled as doesnt make sense for it to be not enabled and then the other values for mongo set.

Signed-off-by: bradmccoydev <bradmccoydev@gmail.com>

### CODE REVIEW

File: content/docs/0.9.x/operate/advanced_install_options/index.md [hunk 1]
```diff
@@ -140,9 +145,25 @@ If you want to use an externally hosted MongoDB instead of the MongoDB installed
 
 ```console
 helm upgrade keptn keptn --install -n keptn --create-namespace
---set=control-plane.mongodb.enabled=false,
+--set=control-plane.mongodb.enabled=true,
```
Comment (mowies) [inline content/docs/0.9.x/operate/advanced_install_options/index.md:148]: Please revert this to `false`. When installing keptn with an external MongoDB we don't want to install our internal MongoDB, so it makes sense that this option is disabled here.

**Review by mowies — CHANGES_REQUESTED** [review]
LGTM except for the comment I left

**Comment by bradmccoydev** [comment]
Thanks reverted back

**Review by mowies — APPROVED** [review]
LGTM 👍🏼

**Comment by mowies** [comment]
Hey @bradmccoydev !
I created a new branch where documentation for the next version of keptn will be prepared (`next-version-preparations`). In there, I created a new folder `content/docs/0.10.x`.

I already switched the target of this PR to point to that branch.
Could you please move you additions to that new subfolder?
Then we can nicely release the docs together with the new features in the next version of keptn.

**Comment by bradmccoydev** [comment]
@mowies Ok, I have done the following
- git remote add upstream git@github.com:keptn/keptn.github.io.git
- git pull upstream next-version-preparations
- Moved new ingress documentation from 0.9.x to 0.10.x
- Run Hugo server and tested all good
- Committed new changes that are showing above

Please let me know if you need me to do anything else, thanks

File: content/docs/0.10.x/operate/advanced_install_options/index.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ The following flags are available:
...
 | `control-plane.prefixPath`                            | Allows to set a prefixPath for the api-gateway. This value has to start with a "/" but must not contain a trailing "/"                            | URI, e.g., /mykeptn                   | empty                          |
 | `control-plane.bridge.cliDownloadLink`                | Specifies a download link for the Keptn CLI which is displayed in Keptn Bridge                                                                    | any string/URL                        | null                           |
-| `control-plane.bridge.lookAndFeelUrl`                 | Specifies a [custom look and feel](https://github.com/keptn/keptn/tree/release-0.10.0/bridge#custom-look-and-feel) for Keptn Bridge                | any URL containing a zip archive      | null                           |
+| `control-plane.bridge.lookAndFeelUrl`                 | Specifies a [custom look and feel](https://github.com/keptn/keptn/tree/release-0.9.0/bridge#custom-look-and-feel) for Keptn Bridge                | any URL containing a zip archive      | null                           |
```
Comment (thisthat) [inline content/docs/0.10.x/operate/advanced_install_options/index.md:24]:
```suggestion
| `control-plane.bridge.lookAndFeelUrl`                 | Specifies a [custom look and feel](https://github.com/keptn/keptn/tree/release-0.10.0/bridge#custom-look-and-feel) for Keptn Bridge                | any URL containing a zip archive      | null                           |
```

File: content/docs/0.10.x/operate/advanced_install_options/index.md [hunk 3]
```diff
@@ -32,27 +32,32 @@ The following flags are available:
...
+| `control-plane.ingress.path`                          | If ingress is set to true you can specify a path                                                           | any string                            | /                          |
+| `control-plane.ingress.pathType`                      | If ingress is set to true you can specify a pathType                                                       | any string                            | Prefix                          |
+| `control-plane.ingress.tls`                           | If ingress is set to true you can optionally specify tls config                                                         | any string                            | empty                          |
 ### Example: Use a LoadBalancer for api-gateway-nginx
 
 ```console
-helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.10.0 --repo=https://storage.googleapis.com/keptn-installer --set=control-plane.apiGatewayNginx.type=LoadBalancer
+helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.9.0 --repo=https://storage.googleapis.com/keptn-installer --set=control-plane.apiGatewayNginx.type=LoadBalancer
```
Comment (thisthat) [inline content/docs/0.10.x/operate/advanced_install_options/index.md:46]:
```suggestion
helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.10.0 --repo=https://storage.googleapis.com/keptn-installer --set=control-plane.apiGatewayNginx.type=LoadBalancer
```

File: content/docs/0.10.x/operate/advanced_install_options/index.md [hunk 4]
```diff
@@ -32,27 +32,32 @@ The following flags are available:
...
 ```
 
 ### Example: Install execution plane for Continuous Delivery use-case
 
 For example, the **Control Plane with the Execution Plane (for Continuous Delivery)** can be installed by the following command:
 ```console
-helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.10.0 --repo=https://storage.googleapis.com/keptn-installer --set=continuous-delivery.enabled=true
+helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.9.0 --repo=https://storage.googleapis.com/keptn-installer --set=continuous-delivery.enabled=true
```
Comment (thisthat) [inline content/docs/0.10.x/operate/advanced_install_options/index.md:53]:
```suggestion
helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.10.0 --repo=https://storage.googleapis.com/keptn-installer --set=continuous-delivery.enabled=true
```

File: content/docs/0.10.x/operate/advanced_install_options/index.md [hunk 5]
```diff
@@ -32,27 +32,32 @@ The following flags are available:
...
 ```
 
 ### Example: Install execution plane for Continuous Delivery use-case and use a LoadBalancer for api-gateway-nginx
 
 For example, the **Control Plane with the Execution Plane (for Continuous Delivery)** and a `LoadBalancer` for exposing Keptn can be installed by the following command:
 ```console
-helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.10.0 --repo=https://storage.googleapis.com/keptn-installer --set=continuous-delivery.enabled=true,control-plane.apiGatewayNginx.type=LoadBalancer
+helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.9.0 --repo=https://storage.googleapis.com/keptn-installer --set=continuous-delivery.enabled=true,control-plane.apiGatewayNginx.type=LoadBalancer
```
Comment (thisthat) [inline content/docs/0.10.x/operate/advanced_install_options/index.md:60]:
```suggestion
helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.10.0 --repo=https://storage.googleapis.com/keptn-installer --set=continuous-delivery.enabled=true,control-plane.apiGatewayNginx.type=LoadBalancer
```

File: content/docs/0.10.x/operate/advanced_install_options/index.md [hunk 6]
```diff
@@ -81,9 +86,9 @@ For convenience, the following script creates this directory and downloads the r
 ```console
 mkdir offline-keptn
 cd offline-keptn
-curl -L https://github.com/keptn/keptn/releases/download/0.10.0/keptn-0.10.0.tgz -o keptn-0.10.0.tgz
-curl -L https://github.com/keptn/keptn/releases/download/0.10.0/helm-service-0.10.0.tgz -o helm-service-0.10.0.tgz
-curl -L https://github.com/keptn/keptn/releases/download/0.10.0/jmeter-service-0.10.0.tgz -o jmeter-service-0.10.0.tgz
+curl -L https://github.com/keptn/keptn/releases/download/0.9.0/keptn-0.9.0.tgz -o keptn-0.9.0.tgz
+curl -L https://github.com/keptn/keptn/releases/download/0.9.0/helm-service-0.9.0.tgz -o helm-service-0.9.0.tgz
+curl -L https://github.com/keptn/keptn/releases/download/0.9.0/jmeter-service-0.9.0.tgz -o jmeter-service-0.9.0.tgz
```
Comment (thisthat) [inline content/docs/0.10.x/operate/advanced_install_options/index.md:91]:
```suggestion
curl -L https://github.com/keptn/keptn/releases/download/0.10.0/keptn-0.9.0.tgz -o keptn-0.10.0.tgz
curl -L https://github.com/keptn/keptn/releases/download/0.10.0/helm-service-0.9.0.tgz -o helm-service-0.10.0.tgz
curl -L https://github.com/keptn/keptn/releases/download/0.10.0/jmeter-service-0.9.0.tgz -o jmeter-service-0.10.0.tgz
```

File: content/docs/0.10.x/operate/advanced_install_options/index.md [hunk 7]
```diff
@@ -100,7 +105,7 @@ For convenience, you can use the following commands to download and execute the
 cd offline-keptn
 curl -L https://raw.githubusercontent.com/keptn/keptn/master/installer/airgapped/pull_and_retag_images.sh -o pull_and_retag_images.sh
 chmod +x pull_and_retag_images.sh
-KEPTN_TAG=0.10.0 ./pull_and_retag_images.sh "your-registry.localhost:5000/"
+KEPTN_TAG=0.9.0 ./pull_and_retag_images.sh "your-registry.localhost:5000/"
```
Comment (thisthat) [inline content/docs/0.10.x/operate/advanced_install_options/index.md:108]:
```suggestion
KEPTN_TAG=0.10.0 ./pull_and_retag_images.sh "your-registry.localhost:5000/"
```

File: content/docs/0.10.x/operate/advanced_install_options/index.md [hunk 8]
```diff
@@ -118,7 +123,7 @@ For convenience, you can use the following commands to download and execute the
 cd offline-keptn
 curl -L https://raw.githubusercontent.com/keptn/keptn/master/installer/airgapped/install_keptn.sh -o install_keptn.sh
 chmod +x install_keptn.sh
-./install_keptn.sh "your-registry.localhost:5000/" keptn-0.10.0.tgz helm-service-0.10.0.tgz jmeter-service-0.10.0.tgz
+./install_keptn.sh "your-registry.localhost:5000/" keptn-0.9.0.tgz helm-service-0.9.0.tgz jmeter-service-0.9.0.tgz
```
Comment (thisthat) [inline content/docs/0.10.x/operate/advanced_install_options/index.md:126]:
```suggestion
./install_keptn.sh "your-registry.localhost:5000/" keptn-0.10.0.tgz helm-service-0.10.0.tgz jmeter-service-0.10.0.tgz
```

File: content/docs/0.10.x/operate/advanced_install_options/index.md [hunk 9]
```diff
@@ -131,7 +136,7 @@ For example, if a user sets `control-plane.prefixPath=/mykeptn` in the Helm inst
 the Keptn API is located under `http://HOSTNAME/mykeptn/api` and the Keptn Bridge is located under `http://HOSTNAME/mykeptn/bridge`:
 
 ```console
-helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.10.0 --repo=https://storage.googleapis.com/keptn-installer --set=control-plane.apiGatewayNginx.type=LoadBalancer,continuous-delivery.enabled=true,control-plane.prefixPath=/mykeptn
+helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.9.0 --repo=https://storage.googleapis.com/keptn-installer --set=control-plane.apiGatewayNginx.type=LoadBalancer,continuous-delivery.enabled=true,control-plane.prefixPath=/mykeptn
```
Comment (thisthat) [inline content/docs/0.10.x/operate/advanced_install_options/index.md:139]:
```suggestion
helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.10.0 --repo=https://storage.googleapis.com/keptn-installer --set=control-plane.apiGatewayNginx.type=LoadBalancer,continuous-delivery.enabled=true,control-plane.prefixPath=/mykeptn
```

File: content/docs/0.10.x/operate/advanced_install_options/index.md [hunk 10]
```diff
@@ -68,11 +73,11 @@ The following artifacts need to be available locally:
...
 
-Download the Helm charts from the [Keptn 0.10.0 release](https://github.com/keptn/keptn/releases/tag/0.10.0):
+Download the Helm charts from the [Keptn 0.9.0 release](https://github.com/keptn/keptn/releases/tag/0.9.0):
 
-* Keptn Control Plane: https://github.com/keptn/keptn/releases/download/0.10.0/keptn-0.10.0.tgz
-* helm-service (if needed): https://github.com/keptn/keptn/releases/download/0.10.0/helm-service-0.10.0.tgz
-* jmeter-service (if needed): https://github.com/keptn/keptn/releases/download/0.10.0/jmeter-service-0.10.0.tgz
+* Keptn Control Plane: https://github.com/keptn/keptn/releases/download/0.9.0/keptn-0.9.0.tgz
+* helm-service (if needed): https://github.com/keptn/keptn/releases/download/0.9.0/helm-service-0.9.0.tgz
+* jmeter-service (if needed): https://github.com/keptn/keptn/releases/download/0.9.0/jmeter-service-0.9.0.tgz
```
Comment (thisthat) [inline content/docs/0.10.x/operate/advanced_install_options/index.md:80]:
```suggestion
* Keptn Control Plane: https://github.com/keptn/keptn/releases/download/0.10.0/keptn-0.10.0.tgz
* helm-service (if needed): https://github.com/keptn/keptn/releases/download/0.10.0/helm-service-0.10.0.tgz
* jmeter-service (if needed): https://github.com/keptn/keptn/releases/download/0.10.0/jmeter-service-0.10.0.tgz
```

File: content/docs/0.10.x/operate/advanced_install_options/index.md [hunk 11]
```diff
@@ -68,11 +73,11 @@ The following artifacts need to be available locally:
 
 **Download Keptn Helm Charts**
 
-Download the Helm charts from the [Keptn 0.10.0 release](https://github.com/keptn/keptn/releases/tag/0.10.0):
+Download the Helm charts from the [Keptn 0.9.0 release](https://github.com/keptn/keptn/releases/tag/0.9.0):
```
Comment (thisthat) [inline content/docs/0.10.x/operate/advanced_install_options/index.md:76]:
```suggestion
Download the Helm charts from the [Keptn 0.10.0 release](https://github.com/keptn/keptn/releases/tag/0.10.0):
```

**Comment by thisthat** [comment]
Hi @bradmccoydev! Thank you for your help! I noticed a few lines which reference the wrong Keptn version. After these lines are fixed, we can merge this PR :shipit:

**Comment by bradmccoydev** [comment]
Hey @thisthat. Great catch! I have made the changes, thank you!

**Comment by mowies** [comment]
Hi @bradmccoydev !
Unofrtunetely there was some internal miscommunication which meant that I told you to rebase onto the wrong branch. If that's fine with you I will fix the issues with branching and folders on your branch directly and then we can merge this PR.

**Comment by bradmccoydev** [comment]
Hey @mowies no problem, sounds good

**Pull request merged**